dev/integration_tests/module_host_with_custom_build/app/src/main/java/io/flutter/add2app/MainActivity.java

```diff
@@ -0,0 +1,14 @@
+package io.flutter.add2app;
+
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import io.flutter.facade.Flutter;
+
+public class MainActivity extends AppCompatActivity {
+
+  @Override
+  protected void onCreate(Bundle savedInstanceState) {
+    super.onCreate(savedInstanceState);
+    setContentView(Flutter.createView(this, getLifecycle(), "route1"));
```

Comment:
Please don't use the facade classes. Use https://github.com/flutter/flutter/wiki/Experimental:-Add-Flutter-Activity instead.

Reply:
Good point on current fragment dependencies on the new embedding.

I think the easiest way to "exercise the engine and VM" which I think is the goal here is to just let MainActivity extend io.flutter.app.FlutterActivity instead of using the facade classes. 
cc @matthew-carroll